[MTM-60743]: Enhance documentation of c8y-docs uploading same CA across tenant (#2408)
SoftwareAG/c8y-docs: content/device-integration/mqtt-bundle/implementation.md

* The platform's trust store cannot be empty. At least one trusted certificate must be uploaded to the platform.
* The device's MQTT client must be configured to not send certificates if it does not find its root certificate in the accepted issuers list returned by the server during handshake. In most cases this happens automatically. It is known that it's not working with the MQTT client and Java 11. However, it works with Java 8.
* In order to support this situation, the platform must be configured accordingly. In case you experience issues please contact [product support](/additional-resources/contacting-support/).

> * If some other tenant uploaded a same certificate the device will always try to authorize itself with certificates but will fail in tenant identification.

Reviewer: Not sure if you have not seen my comment, but this seems to be a duplicate of the sentence below, or is this something different?

Author: `If all of the cases above are met and the device connection is still rejected due to certificates validation, then probably some other tenant uploaded a certificate with the same 'Common Name' as one of those sent by your device. In this case the device will always try to authorize itself with certificates.`

` If some other tenant uploaded a same certificate the device will always try to authorize itself with certificates but will fail in tenant identification.`

Above both sentence is differen as previous one is about same common name (CN) not certificate but below sentence talking about same certificate irrespective of common name (CN).

> * Probably some other tenant uploaded a same certificate. In this case the device will always try to authorize itself with certificates but will fail in tenant identification.

Reviewer: ```suggestion
* If some other tenant uploaded a same certificate the device will always try to authorize itself with certificates but will fail in tenant identification.
```

Reviewer: Isn´t this a duplicate of the following item?
